### PR TITLE
1491 transactions websockets and versioning for transactions

### PR DESCRIPTION
**Pull request by xoscar**
This PR enables the websockets usages from the FE as part of the transaction run page.

## Changes

- Listens to events during the transaction run execution.

## Fixes

- [[Transactions] Websockets and Versioning for Transactions#1491](https://github.com/kubeshop/tracetest/issues/1491)

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by mathnogueira — APPROVED** [review]
BE looks fine

File: server/http/websocket/subscribe.go [hunk 1]
```diff
@@ -62,6 +62,11 @@ func (e subscribeCommandExecutor) ResourceUpdatedEvent(resource interface{}) Eve
 		mapped = e.mappers.Out.Run(&v)
 	case *model.Run:
 		mapped = e.mappers.Out.Run(v)
+	case model.TransactionRun:
+		mapped = e.mappers.Out.TransactionRun(v)
+	case *model.TransactionRun:
+		var value model.TransactionRun = *v
+		mapped = e.mappers.Out.TransactionRun(value)
```
Comment (mathnogueira) [inline server/http/websocket/subscribe.go:69]: Can you use this instead?

```suggestion
		mapped = e.mappers.Out.TransactionRun( *v)
```

Comment (xoscar) [reply server/http/websocket/subscribe.go:69]: I tried that but the TransactionRun function expects a model instance value not a pointer 😢

Comment (schoren) [reply server/http/websocket/subscribe.go:69]: Are you sure you tried the exact code as Matheus suggested? It’s the exact same way we’ve done it for test runs, so it should work

**Pull request merged**